.github/workflows/prerelease.yml

```diff
@@ -63,7 +70,239 @@ jobs:
           files: |
             ./LICENSE
             ./builds/ci/package/*
+#
+#     LLVM Test Suite
+#
+  run_llvm_ts:
+    name: "Run LLVM test suite"
+    needs: build
+    strategy:
+      matrix:
+        llvm-version: [17]
+        image-version: [22.04]
+        vast-target: ['hl', 'llvm', 'bin']
+
+    runs-on: ubuntu-${{ matrix.image-version }}
+    timeout-minutes: 60
+    container:
+      image:
+        ghcr.io/trailofbits/vast-ubuntu-${{ matrix.image-version }}-llvm-${{ matrix.llvm-version }}-dev:latest
+
+    steps:
+     - name: Fetch VAST artifact
+       uses: actions/download-artifact@v4
+       with:
+         name: VAST
+
+     - name: Unpack VAST
+       run: mkdir vast && tar -xf VAST-* -C vast --strip-components=1
+
+     - name: Export vast binaries
+       run: echo "${PWD}/vast/bin/" >> $GITHUB_PATH
+
+     - name: Install test suite dependencies
+       run: |
+          apt-get update
+          apt-get -y install clang
+          pip3 install lit psutil
+
+     - name: Clone test suite repository
+       uses: actions/checkout@v4
+       with:
+        repository: trailofbits/vast-llvm-test-suite
+        ref: main
+        path: llvm-test-suite
+        fetch-depth: 1
+
+     - name: Run test-suite
+       continue-on-error: true
+       run: |
+        mkdir llvm-test-suite/build && cd llvm-test-suite/build/
+        cmake ../ -DCMAKE_BUILD_TYPE=Release \
+          -DCMAKE_C_COMPILER=$(readlink -f ../tools/compiler) \
+          -DCMAKE_CXX_COMPILER=$(readlink -f ../tools/compiler++) \
+          -C=../cmake/caches/O3.cmake \
+          -DTEST_SUITE_COLLECT_CODE_SIZE=OFF \
+          -DTEST_SUITE_SUBDIRS=SingleSource \
+          -DVAST_TEST_SUITE_ENABLE=ON \
+          -DVAST_TEST_SUITE_TARGET=${{ matrix.vast-target }}
+        make -j $(nproc) -k
+
+     - name: Collect run data (mlir)
+       if: matrix.vast-target != 'bin'
+       working-directory: ./llvm-test-suite/build
```

Comment:
why in `build` dir when everything you access is in `../`?

Reply:
The script performs globbing on the results and generates the benchmark names on that. So it would generate incorrect `.json` with different names than expected.